src/shared/ui/badge/badge.tsx

```diff
@@ -0,0 +1,43 @@
+import classNames from 'classnames/bind';
+import Image, { StaticImageData } from 'next/image';
+
+import { Paragraph } from '@/shared/ui/paragraph';
+import { WidgetTitle } from '@/shared/ui/widget-title';
+
+import styles from './badge.module.scss';
+
+type BadgeProps = {
+  title: string;
+  paragraphs: string[];
+  image: StaticImageData;
+  alt: string;
+};
+
+export const Badge = (data: BadgeProps) => {
+  const cx = classNames.bind(styles);
```

Comment:
Please move `cx` outside the component, similar to how it's handled in other components